src/routes.js

```diff
@@ -0,0 +1,131 @@
+import 'babel-core/register';
```

Comment:
@FrancescoCioria consider that `index.js` is used only in production (`yarn start`) but in testing (`yarn test`) I start from `app.js` (`index.js` is not imported at all).